### ISSUE TITLE
cant make pdfs (on some computers)

### ISSUE THREAD

**Issue by Sonderwoods** [interopxyz/PdfPlus#6]
hi David,

on 3 of 5 computers we get this exception, however it works on the last two. any ideas?

![image](https://user-images.githubusercontent.com/19936679/222919659-e4fdccd0-e3e3-4fd4-817a-559bedd54a09.png)

![image](https://user-images.githubusercontent.com/19936679/222919666-31730f37-f157-4661-ad7e-30b6d3e1f7f3.png)

![image](https://user-images.githubusercontent.com/19936679/222919699-8f4f8f29-4a8a-408c-98e3-e44d62872db0.png)

**Comment by Sonderwoods** [comment]
works fine on your build on food4rhino but not when we build ourselves.

Crashes here:  
https://github.com/interopxyz/PdfPlus/blob/0b32ed9027619f9e94ba5ce1fc0f91c046255f6d/PdfPlus/Classes/Page.cs#L169

I tried putting breakpoints inside the render method with no luck.
Doesnt even fire the method.
![image](https://user-images.githubusercontent.com/19936679/222919849-f58c044f-71dc-469d-ab9a-4c18539190dc.png)


I've added all of the projects you uploaded with zip, but only referenced PdfSharp and PdfSharp.Charting.

Should I reference something else instead?
